internal/database/client.go

```diff
@@ -0,0 +1,50 @@
+package database
+
+import (
+	"fmt"
+
+	"github.com/boltdb/bolt"
+)
+
+const (
+	databaseFile = "madprobe.db"
+	probeBucket  = "Probe"
```

Comment:
No need for uppercase here.
```suggestion
	probeBucket  = "probe"
```